commands-extender/print-preview.command.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import { Observable } from "rxjs/Observable";
+
+import { Command, ExecutionContext } from "progress-sitefinity-adminapp-sdk/app/api/v1";
+
+
+/**
+ * Serves as a command that gets invoked when the print preview button
+ * in the actions menu is clicked.
+ */
+@Injectable()
+export class PrintPreviewCommand implements Command {
+
+    /**
+     * Initializes a new instance of the PrintPreviewCommand.
+     * @param router The router that is used to navigate.
+     */
+    constructor(protected router: Router) {
+
+    }
+
+    /**
+     * This method gets invoked when the print preview action is clicked.
```

Comment:
we can use *action command* instead of just *action*, because while reading it one can get confused what is ```Action``` and what is ```Command```. Also why in comments we keep calling it actions and the class is called ```Command```

Reply:
because it is the actions menu

Reply:
so lets call it ```action menu command``` as it ```implements Command```